warehouse/accounts/__init__.py

```diff
@@ -98,6 +98,8 @@ def _authenticate(userid, request):
 
     if user.is_superuser:
         principals.append("group:admins")
+    if user.is_moderator or user.is_superuser:
```

Comment:
With the thought that filtering things to one principal, considering superuser to inherit moderator privileges would mean things could be flagged as moderators.

If someone else wanted to weigh in, it may make sense to not do this in this way though.

Reply:
I think we just need to add moderators to the moderators group. I think we can just set up the ACLs based on whether the user is in the admin group or not.

Reply:
Scratch that, after seeing the implementation this makes sense as-is.